server/config/subject_page_pb2.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: subject_page.proto
-"""Generated protocol buffer code."""
-from google.protobuf.internal import builder as _builder
+
```

Comment:
should there be changes in this file ..?

Reply:
These changes are generated for me whenever I run `./run_server.sh`. Since it's autogenerated it shouldn't matter, but removed these changes anyway.